lib/git_reflow/git_server/git_hub/pull_request.rb

```diff
@@ -63,7 +63,19 @@ def comments
         end
 
         def last_comment
-          "#{comments.last.body.inspect}"
+          if comments.last.nil?
+            ""
+          else
+            "#{comments.last.body.inspect}"
+          end
+        end
+
+        def approved?
```

Comment:
I think the underlying issue here is that we're accounting for a scenario we shouldn't be supporting: `minimum_approvals == 0`.  Git-reflow is built around the idea of building software in teams and ensuring quality control with peer reviews.  I think we should leave this out for now and see if the community finds a need for it later.  In doing so we get much cleaner code here:

```ruby
def approved?
  super and !last_comment.match(self.class.approval_regex).nil?
end
```